ci/cases/pr/C96_S2SWA_gefs_replay_ics.yaml

```
@@ -0,0 +1,19 @@
+experiment:
+  system: gefs
+  mode: forecast-only
+
+arguments:
+  pslot: {{ 'pslot' | getenv }}
+  app: S2SWA
+  resdetatmos: 96
+  resdetocean: 1.0
+  resensatmos: 96
+  nens: 2
+  gfs_cyc: 1
+  start: warm
+  comroot: {{ 'RUNTESTS' | getenv }}/COMROOT
+  expdir: {{ 'RUNTESTS' | getenv }}/EXPDIR
+  idate: 2020110100
+  edate: 2020110100
+  yaml: {{ HOMEgfs }}/ci/cases/yamls/gefs_replay_ci.yaml
+  icsdir: {{ 'TOPICDIR' | getenv }}/REPLAY_ICs/CI
```

This should be `ICSDIR_ROOT`. I do not believe `TOPICDIR` is defined anywhere in global-workflow, which is why the CI test failed on WCOSS2. We'll need to verify that `REPLAY_ICs/CI` is in the following directories on each platform.
Hera:` /scratch1/NCEPDEV/global/glopara/data/ICSDIR`
WCOSS2:` /lfs/h2/emc/global/noscrub/emc.global/data/ICSDIR`
Orion: `/work/noaa/global/glopara/data/ICSDIR`
Hercules: `/work/noaa/global/glopara/data/ICSDIR`
 

```suggestion
  icsdir: {{ 'ICSDIR_ROOT' | getenv }}/REPLAY_ICs/CI
```